packages/Poll/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+export default class Poll extends React.Component {
+  constructor (props) {
+    super(props)
+    if (typeof window !== 'undefined') {
```

Comment:
Why do this in constructor instead of `componentDidMount`?

Reply:
Ah I thought it was deprecated, will switch